src/components/cc-tile-status-codes/cc-tile-status-codes.js

```diff
@@ -80,75 +80,77 @@ export class CcTileStatusCodes extends LitElement {
 
   firstUpdated () {
 
-    this._ctx = this.renderRoot.getElementById('chart');
-    this._chart = new Chart(this._ctx, {
-      type: 'doughnut',
-      options: {
-        responsive: true,
-        maintainAspectRatio: false,
-        plugins: {
-          datalabels: {
-            display: false,
-          },
-          legend: {
-            onClick: function (e, legendItem) {
-              this.chart.data.labels.forEach((label, i) => {
-                const sameLabel = (label === legendItem.text);
-                if (xor(e.native.shiftKey, sameLabel)) {
-                  this.chart.toggleDataVisibility(i);
-                }
-              });
-              this.chart.update();
-            },
-            onHover: (e) => {
-              this._ctx.style.cursor = 'pointer';
-            },
-            onLeave: (e) => {
-              this._ctx.style.cursor = null;
+    if (!this.error) {
```

Comment:
Is there a case for the following syntax?
```js
firstUpdated () {
  if (this.error) {
    return;
  }
  // [initial function content]
}
```
Spares an nested block and gives a more legible diff... :hugs: